README.md

```diff
@@ -1059,14 +1059,18 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [trayhost](https://github.com/shurcooL/trayhost) - Cross-platform Go library to place an icon in the host operating system's taskbar.
 * [zenity](https://github.com/ncruces/zenity) - Cross-platform Go library and CLI to create simple dialogs that interact graphically with the user.
 
-
 **[⬆ back to top](#contents)**
 
 ## Hardware
 
 *Libraries, tools, and tutorials for interacting with hardware.*
 
-See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive list.
+* [emgo](https://github.com/ziutek/emgo) - Go-like language for programming embedded systems (e.g. STM32 MCU).
+* [ghw](https://github.com/jaypipes/ghw) - Golang hardware discovery/inspection library.
+* [go-rpio](https://github.com/stianeikeland/go-rpio) - GPIO for Go, doesn't require cgo.
+* [go-osc](https://github.com/hypebeast/go-osc) - Open Sound Control (OSC) bindings for Go.
+* [joystick](https://github.com/SimulatedSimian/joystick) - a polled API to read the state of an attached joystick.
```

Comment:
https://github.com/SimulatedSimian/joystick seems to be a fork and is 5 commits behind its upstream repo.
This can be changed to https://github.com/0xcafed00d/joystick, I think.